frontend/src/components/form/base/ETextField.vue

```diff
@@ -45,46 +45,6 @@ export default {
       default: 'text',
     },
   },
-  data() {
-    return {
-      preventValidationOnBlur: false,
-    }
-  },
-  computed: {
-    inputListeners: function () {
-      const vm = this
-      return Object.assign(
-        {},
-        // attach all $parent listeners
-        this.$listeners,
-
-        // override @input listener for correct handling of numeric values
-        {
-          input: function (value) {
-            vm.$data.preventValidationOnBlur = false
-            if (vm.type === 'number') {
-              vm.$emit('input', parseFloat(value))
-            } else {
-              vm.$emit('input', value)
-            }
-          },
-          blur: function () {
```

Comment:
This code was added in https://github.com/ecamp/ecamp3/pull/3084 and has nothing to do with float handling. Please add back.

Reply:
I think this is not the "right" way to solve this problem and can cause side effects. I'll create a followup issue.

Reply:
#3893